js-intro.js

```diff
@@ -36,8 +62,17 @@ var myNumbers2 = [8, -7, 0, 6, 2]
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
+removeVowel = (str) => {
+  let newArr1 = str.split("")
+  let newArr2 = newArr1.filter(value => value.toLowerCase() !== "a" && value.toLowerCase() !== "e" && value.toLowerCase() !== "i" && value.toLowerCase() !== "o" && value.toLowerCase() !== "u")
```

Comment:
You can also append `.join("")` to the end of this line and just return this variable instead.